File: 0242-valid-anagram/0242-valid-anagram.java
import java.util.Arrays;
class Solution {
    public static boolean isAnagram(String s, String t) {
        if (s.length() == t.length()){
            char[] sCharArray = new char[s.length()];
            char[] tCharArray = new char[t.length()];
            for (int i=0;i<s.length();i++){
                sCharArray[i] = s.charAt(i);
                tCharArray[i] = t.charAt(i);
            }
            Arrays.sort(sCharArray);
            Arrays.sort(tCharArray);
            
            for (int i=0;i<sCharArray.length;i++){
                if (sCharArray[i] != tCharArray[i])
                    return false;
            }
            return true;
        }
        else
            return false;
    }
}

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        charTracker = {}
        second = {}
        for char in s:
            if char not in charTracker:
                charTracker[char] = 1
            else:
                charTracker[char] += 1
        
        for char in t:
            if char not in second:
                second[char] = 1
            else:
                second[char] += 1
        return charTracker == second